Documentation/op-guide/performance.md

```diff
@@ -10,4 +10,40 @@ There are other sub-systems which impact the overall performance of etcd. Each s
 
 ## Benchmarks
 
-TODO
\ No newline at end of file
+To benchmark etcd performance, use [benchmark](https://github.com/coreos/etcd/tree/master/tools/benchmark) CLI tool. Following results are from a 3 member etcd cluster with the hardware configuration:
```

Comment:
Benchmarking etcd performance can be done with the [benchmark](https://github.com/coreos/etcd/tree/master/tools/benchmark) CLI tool included with etcd.

For some baseline performance numbers, we consider a three member etcd cluster with the following hardware configuration: